src/cookies.d.ts causes the compile error TS2307 if we import in TypeScript external modules.

If we import in TypeScript external module,  src/cookies.d.ts causes the compile error TS2307.

## envionment

- TypeScript 1.4.1
- TypeScript 1.5.0-beta

## testcase
External module style of TypeScript

```typescript
/// <reference path="./node_modules/cookies-js/src/cookies.d.ts"/>
import Cookies = require('cookies-js');

var bar = Cookies.get('test');
```

## result

```sh
$ tsc test.ts --module commonjs
test.ts(3,26): error TS2307: Cannot find external module 'cookies-js'.
```

--- a/src/cookies.d.ts
+++ b/src/cookies.d.ts
@@ -27,3 +27,7 @@ declare var Cookies:CookiesStatic;
 declare module "cookies" {
     export = Cookies;
 }
+
+declare module "cookies-js" {
+    export = Cookies;
+}

